web/src/beta/components/TabMenu/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC, useState, ReactNode } from "react";
+
+import Icon from "@reearth/beta/components/Icon";
+import Icons from "@reearth/beta/components/Icon/icons";
+import { styled } from "@reearth/services/theme";
+
+interface TabObject {
+  icon: keyof typeof Icons;
+  component: ReactNode;
+}
+
+type Props = {
+  title: string;
+  tabs: Record<string, TabObject>;
+};
+
+const TabMenu: FC<Props> = ({ tabs, title }) => {
+  const [selected, setSelected] = useState<string | null>(null);
+
+  return (
+    <Wrapper>
+      <Title>{title}</Title>
```

Comment:
For text we have a Text component in the beta/components/ directory. Please use that

Reply:
Done. Thanks :)